### ISSUE TITLE
Can't compile latest master on OSX

### ISSUE THREAD

**Issue by dave-tucker** [cuberite/cuberite#2684]
```
Scanning dependencies of target OSSupport
[ 45%] Building CXX object src/OSSupport/CMakeFiles/OSSupport.dir/StackTrace.cpp.o
/Users/dave/dev/cuberite/src/OSSupport/StackTrace.cpp:21:10: error: macro is not used [-Werror,-Wunused-macros]
        #define btsize int
                ^
1 error generated.
make[2]: *** [src/OSSupport/CMakeFiles/OSSupport.dir/StackTrace.cpp.o] Error 1
make[1]: *** [src/OSSupport/CMakeFiles/OSSupport.dir/all] Error 2
make: *** [all] Error 2
```

Looks like some of the fixes for MUSL may have caused this :panda_face: 
https://github.com/cuberite/cuberite/pull/2661

**Comment by dave-tucker** [comment]
Note: I've tried with both GCC and Clang with the same result

**Comment by SphinxC0re** [comment]
Is this still an issue?

**Comment by worktycho** [comment]
Should be fixed by bfe27fb68c8f9e1a087ef47f29e57d9a3e289e21
